conductor_api/src/config.rs

```diff
@@ -351,6 +354,38 @@ impl Configuration {
 
         self
     }
+
+    /// This function checks if there is duplicated file storage from the instances section of a provided
+    /// TOML configuration file. For efficiency purposes, we short-circuit on the first encounter of a
+    /// duplicated values.
+    fn check_instances_storage(&self) -> Result<(), String> {
```

Comment:
This function seems misplaced in `ConductorAdmin`. I think it should be in `Config` and called in `Config::consistency_check`.

Reply:
I'm not sure about what you mean.
This is already in the `impl Configuration` block. Do you mean as a free function?
Or in a different directory?